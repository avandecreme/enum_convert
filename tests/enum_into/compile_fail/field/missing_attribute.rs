use enum_convert::EnumInto;

#[derive(EnumInto)]
#[enum_into(Target)]
enum Source {
    #[enum_into(Target::Data)]
    Stuff {
        // Missing #[enum_into(Target::a)]
        x: i32,
    },
}

enum Target {
    Data { a: i64 },
}

fn main() {}
