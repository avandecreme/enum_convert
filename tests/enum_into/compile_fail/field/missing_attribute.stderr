error[E0559]: variant `Target::Data` has no field named `x`
 --> tests/enum_into/compile_fail/field/missing_attribute.rs:9:9
  |
9 |         x: i32,
  |         ^ `Target::Data` does not have this field
  |
  = note: all struct fields are already assigned
